rpc/management/commands/purge.py

```diff
@@ -28,6 +39,10 @@ def handle(self, *args, **options):
 
         Assignment.objects.all().delete()
         ActionHolder.objects.all().delete()
+        HistoricalRfcToBeLabel.objects.all().delete()
+        HistoricalRfcToBe.objects.all().delete()
+        HistoricalLabel.objects.all().delete()
+        RpcDocumentComment.objects.all().delete()
```

Comment:
I don't object to this, but I still think it should be happening automatically via database configuration (on_delete).
Or is there a reason to keep the history after you delete an object?

Reply:
I'm not sure that works? Are we doing it somewhere? Looking at the rpc database,
```
rpctools=# \d rpc_historicalrfctobe
                                     Table "public.rpc_historicalrfctobe"
          Column          |           Type           | Collation | Nullable |             Default
[...]
Foreign-key constraints:
    "rpc_historicalrfctob_history_user_id_5e1aa891_fk_rpcauth_u" FOREIGN KEY (history_user_id) REFERENCES rpcauth_user(id) DEFERRABLE INITIALLY DEFERRED
```
i.e., there's no foreign key pointing to the `rpc_rfctobe` table that would trigger a cascade.

The history tracks deleted records, so in general at least the history tables have a different lifecycle from the records they describe.

Reply:
From our offline discussion, for posterity the cascaded deletion is [the cascade_delete_history option](https://django-simple-history.readthedocs.io/en/latest/historical_model.html#deleting-historical-record)

Reply:
For now let's be explicit about deleting history records.